developer/src/kmc-kmn/src/compiler/kmn-compiler-messages.ts

```diff
@@ -79,649 +79,644 @@ export class KmnCompilerMessages {
   // kmc-kmn, which would avoid a number of legacy issues. Questions about
   // parameterisation.
 
-  /** @internal */
-  static Fatal_UnexpectedException = (o:{e: any}) => m(this.FATAL_UnexpectedException, null, o.e ?? 'unknown error');
-  /**
-   * Raised when KmnCompiler or one of its components experiences an internal
-   * error. If you experience this error, it should be reported to the Keyman
-   * team for resolution via https://github.com/keymanapp/keyman/issues/new
-   */
   static FATAL_UnexpectedException = SevFatal | 0x900;
+  static Fatal_UnexpectedException = (o:{e: any}) => CompilerMessageSpecWithException(
+    this.FATAL_UnexpectedException,
+    null,
+    o.e ?? 'unknown error',
+    `Raised when KmnCompiler or one of its components hasan internal
```

Comment:
```suggestion
    `Raised when KmnCompiler or one of its components has an internal
```

Reply:
Fixed in #10888: https://github.com/keymanapp/keyman/pull/10888/files#diff-98f78757536605690494383d67629fd5d53ff60be3df468d73ae5d20186e578cR87